This fixes the following error:
```
[cmake] CMake Error at /lib64/cmake/glm/glmConfig.cmake:84 (message):
[cmake]   The imported target "glm::glm" references the file
[cmake] 
[cmake]      "/usr/lib/libglm.a"
[cmake] 
[cmake]   but this file does not exist.  Possible reasons include:
[cmake] 
[cmake]   * The file was deleted, renamed, or moved to another location.
[cmake] 
[cmake]   * An install or uninstall procedure did not complete successfully.
[cmake] 
[cmake]   * The installation package was faulty and contained
[cmake] 
[cmake]      "/lib64/cmake/glm/glmConfig.cmake"
[cmake] 
[cmake]   but not all the files it references.
[cmake] 
[cmake] Call Stack (most recent call first):
[cmake]   CMakeLists.txt:11 (find_package)
[cmake] 
[cmake] 
```

--- a/runtime-scientific/glm/autobuild/defines
+++ b/runtime-scientific/glm/autobuild/defines
@@ -14,4 +14,5 @@ ABTYPE=cmakeninja
 CMAKE_AFTER="-DCMAKE_BUILD_TYPE=Release \
              -DCMAKE_INSTALL_PREFIX=/usr \
              -DCMAKE_INSTALL_DATAROOTDIR=/usr/lib/cmake \
+             -DGLM_BUILD_LIBRARY=OFF \
              -W no-dev"

--- a/runtime-scientific/glm/spec
+++ b/runtime-scientific/glm/spec
@@ -1,4 +1,5 @@
 VER=1.0.1
+REL=1
 SRCS="git::commit=tags/$VER::https://github.com/g-truc/glm"
 CHKSUMS="SKIP"
 CHKUPDATE="anitya::id=1181"

